Implements CSV lists for dwarf-op (#423)
DFHack/scripts: changelog.txt

- `gui/workflow`: restore functionality to the add/remove/order hotkeys on the workflow status screen
- `emigration`: fix emigrant logic so unhappy dwarves leave as designed
- `gui/gm-unit`: allow ``+`` and ``-`` to adjust skill values as intended instead of letting the filter intercept the characters

> - `dwarf-op`: fixed a crash in dwarf-op caused by a typo in the dorf_tables file

Reviewer: I try to use "crash" for full crashes/segfaults only:
```suggestion
- `dwarf-op`: fixed an error in dwarf-op caused by a typo in the dorf_tables file
```

> - `dwarf-op`: fixed an error in dwarf-op caused by a typo in the dorf_tables file

Reviewer 2: could you be more specific here? What works now when it didn't before? Like, which profession/job value?

Author: Fisherman was in as fisher. I can add that to the log, but a user wouldn't have known what was happening without heavy debugging

Author: I suppose maybe something like:
`fixed an error in dwarf-op preventing it from optimizing dwarves, sometimes`
I dunno, that doesn't really seem any better. I could just go extra long:
```
fixed an error in dwarf-op caused by a typo in the dorf_tables file. This typo (FISHER, instead of FISHERMAN) caused the script to error when creating miners
```

Reviewer 2: Would something like "fixed application of MINER job attributes" be accurate?

Author: Nearly, it's not the application of it that was messed up. It was data inside of it. It referenced something that didn't exist.

Reviewer 2: from a user perspective, what action did it prevent from working? Which feature was broken?

Author: The miner job. So anything involving it. The optimize feature or applyjobs with miner as one of them. And not even all the time, cause the profession doesn't have to be used.. Although the more miners, the more exponentially likely it is to be used.

I think of it as a linker error, except during runtime. With the unit test it turns that kind of issue into a compile time error (sort of.. Build time? Test time? Lol)

Reviewer 2: So how about "fixed application of Miner job to dwarves"?

Author: Yea it is accurate, since the Miner job isn't part of the dwarf-op script.. So naming it specifically I think implies a problem with its definition.

> - `dwarf-op`: fixed an error in dwarf-op caused by a typo in the dorf_tables file

Author: ```suggestion
- `dwarf-op`: fixed error when applying the Miner job to dwarves
```
